doc/governance.rst

```diff
@@ -0,0 +1,130 @@
+
+.. _governance:
+
+===========================================
+Scikit-learn governance and decision-making
+===========================================
+
+The purpose of this document is to formalize the governance process used by the
+scikit-learn project, to clarify how decisions are made and how the various
+elements of our community interact.
+This document establishes a decision-making structure that takes into account
+feedback from all members of the community and strives to find consensus, while
+avoiding any deadlocks.
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
```

Comment:
minor nitpick: there should probably be a comma after the word "design"

Reply:
the Oxford comma...